workflow_tests/cla.test.ts

```diff
@@ -60,13 +71,17 @@ describe('test workflow cla', () => {
                 },
             };
             it('workflow executes, CLA assistant step not run', async () => {
-                const repoPath = mockGithub.repo.getPath('testClaWorkflowRepo') || '';
-                const workflowPath = path.join(repoPath, '.github', 'workflows', 'cla.yml');
-                let act = new eAct.ExtendedAct(repoPath, workflowPath);
+                const repoPath: string = mockGithub?.repo.getPath('testClaWorkflowRepo') ?? '';
+                const workflowPath: string = path.join(repoPath, '.github', 'workflows', 'cla.yml');
+                let act: eAct.ExtendedAct = new eAct.ExtendedAct(repoPath, workflowPath);
```

Comment:
Type is inferred in these cases I think
```suggestion
                const repoPath = mockGithub?.repo.getPath('testClaWorkflowRepo') ?? '';
                const workflowPath = path.join(repoPath, '.github', 'workflows', 'cla.yml');
                let act = new eAct.ExtendedAct(repoPath, workflowPath);
```